devops/scripts/benchmarks/benches/benchdnn.py

```diff
@@ -0,0 +1,190 @@
+# Copyright (C) 2025 Intel Corporation
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from pathlib import Path
+from .base import Suite, Benchmark
+from options import options
+from utils.utils import git_clone, run, create_build_path
+from utils.result import Result
+from utils.oneapi import get_oneapi
+
+
+class OneDnnGraphBench(Suite):
+    def git_url(self):
+        return "https://github.com/oneapi-src/oneDNN.git"
```

Comment:
This redirects to https://github.com/uxlfoundation/oneDNN - perhaps this link should be used directly here?